client/common/js/components/Search.jsx

```diff
@@ -97,28 +119,14 @@ export default function Search({ data = [] }) {
 							className="search-dropdown--tag"
 							onClick={() => updateSelectedTag(tag)}
 							onKeyDown={handleArrowKeys}
-							tabIndex="0"
+							tabIndex="-1"
```

Comment:
Instead of adding a tabIndex=-1, can we just use a different HTML structure? I am thinking `<ul><li role="option">`

Reply:
Unless I'm missing something, doing this still makes the buttons tabbable?

Reply:
After discussing with @dkaoster I think it's fine to keep it as is. I was thinking of removing the button element and just using li, but he pointed out that might be more fiddly than needed.